### PR TITLE
Monika_GLaD

### PR DESCRIPTION
**Pull request by SuperNova2810**
Anyway this is for Monika and our beloved GLaDOS the most famous testing 'n killing machine!
You may change a couple of things, I don't know how to input answers, and feel free to put a couple of "Yes" and No" once in a while!
And every other change feel free too because, I'm really newbie on this thingy!
Is funny! Heh, have a great day!
By the way, for some reason a part of the Monika_God is in the end, delete it please.

### CODE REVIEW

**Comment by kaido1224** [comment]
uh, i think it's up to you to do the formatting correctly.

File: Monika After Story/game/script-topics.rpy [hunk 1]
```diff
@@ -56,7 +56,48 @@ init 11 python:
...
+                                                            unlocked=False,
+                                                            pool=False,
+                                                            conditional=None,
+                                                            action=None,
+                                                            start_date=None,
+                                                            end_date=None,
+                                                            unlock_date=None
+                                                            ))                                              
+label Monika_GLaDOS:                                                
+    m 1a "[player], have you ever played the "Portal series?"
```
Comment (aldoram5) [inline Monika After Story/game/script-topics.rpy]: This is the line with the first double quotes issue I'd recommend:
    m 1a "[player], have you ever played the Portal series?"

File: Monika After Story/game/script-topics.rpy [hunk 2]
```diff
@@ -56,7 +56,48 @@ init 11 python:
...
+    m 1f "Besides..."
+    m 1g "Dan Salvato once spoke about it on Twitter!"
+    m 1h "By the way you already follow me right?"
+    m 1i "Anyway, [player], I really like it's both concepts, both GLaDOS and the Portal mechanics"
+    m 1j "Would you become like GLaDOS for me? Not the testing dummy, but... immortal?"
+    m 1k "God, I really hope you can live that far, and then, my God, we'll be forever, together!"
+    m 1l "Doesn't that sound just like me?"
+    m 1m "Gorgeous!"
+    m 1n "Anyway, I really hope for that, don't you, and to be honest, I really like GLaDOS herself"
+    m 1o "Not the 'kill everyone part', a human inside a machine, a lot of people would do that,
```
Comment (aldoram5) [inline Monika After Story/game/script-topics.rpy]: Here you're missing the closing double quote, it should be:
    m 1o "Not the 'kill everyone part', a human inside a machine, a lot of people would do that,"

**Comment by ThePotatoGuy** [comment]
okay this is all wrong. Please add new topics to the **bottom** of `script-topics`.

**Pull request closed without merging**